src/errors/runtime/templates.js

```diff
@@ -7,101 +7,100 @@ import { RUNTIME_ERRORS } from '../types';
                                           '{errMessage}',
 
-    [RUNTIME_ERRORS.requestHookConfigureAPIError]: 'There was an error while configuring the request hook:\n' +
+    [RUNTIME_ERRORS.requestHookConfigureAPIError]: 'Attempt to configure a request hook resulted in the following error:\n' +
                                                    '\n' +
                                                    '{requestHookName}: {errMsg}',
 
     [RUNTIME_ERRORS.forbiddenCharatersInScreenshotPath]: 'There are forbidden characters in the "{screenshotPath}" {screenshotPathType}:\n' +
                                                          ' {forbiddenCharsDescription}',
 
-    [RUNTIME_ERRORS.cannotFindFFMPEG]: 'Unable to locate the FFmpeg executable required to record videos. Do one of the following:\n' +
+    [RUNTIME_ERRORS.cannotFindFFMPEG]: 'TestCafe cannot record videos because it cannot locate the FFmpeg executable. Try one of the following solutions:\n' +
                                        '\n' +
-                                       '* add the FFmpeg installation directory to the PATH environment variable,\n' +
-                                       '* specify the path to the FFmpeg executable in the FFMPEG_PATH environment variable or the ffmpegPath video option,\n' +
-                                       '* install the @ffmpeg-installer/ffmpeg package from npm.',
+                                       '* add the path of the FFmpeg installation directory to the PATH environment variable,\n' +
+                                       '* specify the path of the FFmpeg executable in the FFMPEG_PATH environment variable or the ffmpegPath option,\n' +
+                                       '* install the @ffmpeg-installer/ffmpeg npm package.',
 
-    [RUNTIME_ERRORS.cannotFindTypescriptConfigurationFile]:            'Unable to find the TypeScript configuration file in "{filePath}"',
-    [RUNTIME_ERRORS.clientScriptInitializerIsNotSpecified]:            'Specify the JavaScript file path, module name or script content to inject a client script.',
+    [RUNTIME_ERRORS.cannotFindTypescriptConfigurationFile]:            '"{filePath}" does not contain a TypeScript configuration file.',
+    [RUNTIME_ERRORS.clientScriptInitializerIsNotSpecified]:            'Initialize your clientScript with one of the following: a JavaScript script, a JavaScript file path, or the name of a JavaScript module.',
     [RUNTIME_ERRORS.clientScriptBasePathIsNotSpecified]:               'Specify the base path for the client script file.',
-    [RUNTIME_ERRORS.clientScriptInitializerMultipleContentSources]:    'You cannot combine the file path, module name and script content when you specify a client script to inject.',
+    [RUNTIME_ERRORS.clientScriptInitializerMultipleContentSources]:    'Client scripts can only have one initializer: a JavaScript script, a JavaScript file path, or the name of a JavaScript module.',
```

Comment:
I don't like "JavaScript script" :-) How about this?  

```suggestion
    [RUNTIME_ERRORS.clientScriptInitializerMultipleContentSources]:    'Client scripts can only have one initializer: JavaScript code, a JavaScript file path, or the name of a JavaScript module.',
```

Reply:
lgtm! :)